models/cryptopunks/ethereum/cryptopunks_ethereum_floor_price_over_time.sql

```diff
@@ -0,0 +1,126 @@
+{{ config(
+        alias ='floor_price_over_time',
+        unique_key='day',
+        post_hook='{{ expose_spells(\'["ethereum"]\',
+                                    "project",
+                                    "cryptopunks",
+                                    \'["cat"]\') }}'
+        )
+}}
+
+with all_listings as (
+    select  `punkIndex` as punk_id
+            , 'Listing' as event_type
+            , case when `toAddress` = '0x0000000000000000000000000000000000000000' then 'Public Listing'
+                    else 'Private Listing'
+                end as event_sub_type
+            , `minValue`/1e18 as listed_price
+            , `toAddress` as listing_offered_to 
+            , evt_block_number
+            , evt_index
+            , evt_block_time
+            , evt_tx_hash
+    from {{ source('cryptopunks_ethereum','CryptoPunksMarket_evt_PunkOffered') }}
+)
+, all_no_longer_for_sale_events (
+    select  `punkIndex` as punk_id
+            , 'No Longer For Sale' as event_type
+            , case when evt_tx_hash in (select evt_tx_hash from cryptopunks_ethereum.CryptoPunksMarket_evt_PunkBought) then 'Punk Bought'
```

Comment:
can you use source() syntax here?

Reply:
Ah yes good catch thank you! Just fixed (+ also on the listings over time file)